en/flight_controller/omnibus_f4_sd.md

```diff
@@ -0,0 +1,123 @@
+# Omnibus F4 SD
+
+The *Omnibus F4 SD* is a controller board designed for racers. 
+In contrast to a typical racer board it has some additional features, such as an SD card and a faster CPU.
+
+These are the main differences compared to a [Pixracer](../flight_controller/pixracer.md):
+- Lower price
+- Fewer IO ports (though it's still possible to attach a GPS or a Flow sensor for example)
+- Less RAM (192 KB vs. 256 KB) and FLASH (1 MB vs. 2 MB)
+- Same board dimensions as a *Pixracer*, but slightly smaller form factor (because it has less connectors)
+
+> **Tip** All the usual PX4 features can still be used for your racer!
```

Comment:
FYI slightly modified this to make it clear that you can still use PX4 features. We could instead state what these limitations mean - fewer concurrently connected peripherals, there isn't much room for additional software in flash or RAM. I figure that the racer audience doesn't care about this.